scripts/predicts.py

```diff
@@ -0,0 +1,109 @@
+                                ("Sampling_effort", ("float")),
+                                ("Rescaled_sampling_effort", ("float")),
+                                ("Habitat_as_described", ("char")),
+                                ("Predominant_land_use", ("char")),
+                                ("Source_for_predominant_land_use", ("char")),
+                                ("Use_intensity", ("char")),
+                                ("Km_to_nearest_edge_of_habitat", ("float")),
+                                ("Years_since_fragmentation_or_conversion", ("float")),
+                                ("Transect_details", ("char")),
+                                ("Coordinates_method", ("char")),
+                                ("Longitude", ("float")),
+                                ("Latitude", ("float")),
+                                ("Country_distance_metres", ("int")),
+                                ("Country", ("char")),
+                                ("UN_subregion", ("char")),
+                                ("UN_region", ("char")),
+                                ("Ecoregion_distance_metres", ("float")),
+                                ("Ecoregion", ("char")),
+                                ("Biome", ("char")),
+                                ("Realm", ("char")),
+                                ("Hotspot", ("char")),
+                                ("Wilderness_area", ("char")),
+                                ("N_samples", ("int")),
+                                ("Taxon_number", ("int")),
+                                ("Taxon_name_entered", ("char")),
+                                ("Indication", ("char")),
+                                ("Parsed_name", ("char")),
+                                ("Taxon", ("char")),
+                                ("COL_ID", ("int")),
+                                ("Name_status", ("char")),
+                                ("Rank", ("char")),
+                                ("Kingdom", ("char")),
+                                ("Phylum", ("char")),
+                                ("Class", ("char")),
+                                ("Order", ("char")),
+                                ("Family", ("char")),
+                                ("Genus", ("char")),
+                                ("Species", ("char")),
+                                ("Best_guess_binomial", ("char")),
+                                ("Higher_taxa", ("char")),
+                                ("Higher_taxon", ("char")),
+                                ("Measurement", ("float")),
+                                ("Effort_corrected_measurement", ("float"))]
+
+            engine.table = table
+            if not os.path.isfile(engine.format_filename(filename)):
+                engine.download_files_from_archive(self.urls[tablename], [filename], filetype="zip", archivename="predicts_latest_" + str(tablename))
```

Comment:
Since we only have one table we can keep this simple:

`"predicts_latest_" + str(tablename)"` -> `"predicts_data"`